python/ql/src/Expressions/IsComparisons.qll

```diff
@@ -108,14 +108,14 @@ predicate invalid_portable_is_comparison(Compare comp, Cmpop op, ClassValue cls)
     )
     and
     /* OK to use 'is' when comparing items from a known set of objects */
```

Comment:
The autoformatter will expand these comments to take up three lines, so you'll probably want to change them to one-line comments before running it (ditto below).